### PR TITLE
add: Turkish (tr_TR) localization added.

### PR DESCRIPTION
**Pull request by ramazansancar**
upd: pt_BR missing keywords #573 updated
upd: English custom validation attributes added.
upd: package.json updated.

## 📃 Description

A short description of the pull request changes should go here and the sections below should list in detail all changes. You can remove the sections you don't need.

### ➕ Added

- one
- two

### ✏️ Changed

- one
- two

### 🔧 Fixed

- one
- two

### 🗑️ Removed

- one
- two

## 📷 Screenshots

If this PR has any UI/UX changes it's strongly suggested you add screenshots here.

### CODE REVIEW

**Comment by alexjustesen** [comment]
@ramazansancar small change regarding the npm files, can you please remove those changes from this PR? I like to avoid multiple changes where possible especially when they're unrelated.

File: package.json [hunk 1]
```diff
@@ -1,10 +1,26 @@
 {
```
Comment (alexjustesen) [inline package.json:1]: please revert the changes to `package.json` and `package-lock.json`

**Review by alexjustesen — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ramazansancar) [reply package.json:1]: I edited it to make the repo more understandable. My edit to package-lock is again only about the repo information, but I can revert it if you want. No problem. @alexjustesen

**Review by alexjustesen — DISMISSED** [review]

**Comment by alexjustesen** [comment]
@ramazansancar looks like it threw a lint error, please make a fix and I'll merge.

**Comment by ramazansancar** [comment]
@alexjustesen fixed

**Review by alexjustesen — APPROVED** [review]
